examples/onnx/cnnonnx2singa.py

```diff
@@ -105,19 +105,13 @@ def accuracy(pred, target):
 
 
     model = onnx.load('cnn.onnx')
-    rep = sonnx.BackendRep(model,dev)
-    #####backend run multiple times
+    rep = sonnx.prepare(model, dev)
     print('finish init')
     autograd.training = True
     # training process
     for epoch in range(1):
         inputs = tensor.Tensor(device=dev, data=x_train[0:100], stores_grad=False)
```

Comment:
input tensor should have a name.

Reply:
if input order here is the same as input order in to_onnx_model, default name system works